ucm2/Intel/sof-essx8336/HiFi.conf

```diff
@@ -1,71 +1,31 @@
 SectionVerb {
 	EnableSequence [
-		disdevall ""
+		#disdevall ""
 		# Disable all inputs / outputs
 		#   (may be duplicated with disdevall)
-		cset "name='Headphone Switch' off"
-		cset "name='Headset Mic Switch' off"
-		cset "name='Internal Mic Switch' off"
-		cset "name='DAC Mono Mix Switch' off"
+		cset "name='Left Headphone Mixer Left DAC Switch' on"
+		cset "name='Right Headphone Mixer Right DAC Switch' on"
 	]
 }
 
-If.amic {
```

Comment:
Do not move amic outside If blocks. According alsa-info dump, there are no Components defined for this card thus this analog mic block should be activated.